app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -111,4 +111,9 @@
             android:id="@+id/action_navigation_products_to_productsDetailsFragment"
             app:destination="@id/productsDetailsFragment" />
     </fragment>
+    <fragment
+        android:id="@+id/ordersFragment"
+        android:name="com.iti.android.team1.ebuy.ui.orders.view.OrdersFragment"
+        android:label="fragment_orders"
```

Comment:
good work Ahmed 💪, you just forgat to add a label in the strings.xml file

Reply:
ok

Reply:
Done